CMake/kwiver-depends.cmake

```diff
@@ -31,4 +31,8 @@ if(KWIVER_ENABLE_ARROWS)
   include( kwiver-depends-Burn-Out )
   include( kwiver-depends-uuid )
   include( kwiver-depends-kpf )
+  if(KWIVER_ENABLE_SVM)
+    include( kwiver-depends-Boost )
+    include( kwiver-depends-SVM )
```

Comment:
Why is depends Boost in with the SVM one?

FWIW, we're trying and remove the dependency from KWIVER, is it possible to refactor it easily to use KWSys instead?

For consistency, you shouldn't have ENABLE_SVM at this level, if you look at all of the other -depends-* they are always added and have the option( KWIVER-ENABLE-*) inside of them

Reply:
>FWIW, we're trying and remove the dependency from KWIVER, is it possible to refactor it easily to >use KWSys instead?

I don't follow.